public/pages/Overview/containers/Overview/Overview.tsx

```diff
@@ -104,11 +107,30 @@ export default class Overview extends Component<OverviewProps, OverviewState> {
     ];
   }
 
+  onTimeChange = ({ start, end }: { start: string; end: string }) => {};
+
```

Comment:
assuming line 110 is a placeholder for future implementation

Reply:
Yes